docroot/themes/contrib/civictheme/civictheme_library/components/00-base/collapsible/collapsible.js

```diff
@@ -217,7 +217,7 @@ CivicThemeCollapsible.prototype.keydownEvent = function (e) {
 
     // Space.
     if (e.which === 32) {
-      this.dispatchEvent(new CustomEvent('ct.collapsible.toggle', { bubbles: true }));
+      e.target.click();
```

Comment:
@joshua-salsadigital 
we cannot call a specific event such as 'click' because it can be re-bound to do something else and this functionality will be affected.

can you please look into solving this differently. perhaps - having another `if (e.which === 13) {` for `Enter` key

Reply:
The enter keys works same as click. (no change required there)
The space bar explicitly needs to work as if an enter or click was done. 
Hence `e.which === 32` needs to act as if enter or click was triggered on the element.